x-pack/plugins/actions/server/builtin_action_types/lib/send_email.ts

```diff
@@ -33,6 +33,13 @@ export interface Transport {
   host?: string;
   port?: number;
   secure?: boolean; // see: https://nodemailer.com/smtp/#tls-options
+  accessToken?: string;
```

Comment:
Not sure if there's any big advantage, but we could create a top-level `oauth` property here, to hold all these oauth-related properties.  Would at least keep them grouped together.